### DISCUSSION TITLE
Height of lines in pcmanfm-qt

### DISCUSSION THREAD

**Discussion by romildo** [tsujan/Kvantum#917] [Q&A]
Please, how to modify a Kvantum theme in order to get smaller heights for the lines showing the files in pcmanfm-qt?

**Comment by tsujan** [comment]
All parameters are explained in the file `doc/Theme-Config` and its PDF counterpart `doc/Theme-Config.pdf` inside the source.

If you mean pcmanfm-qt's list view, it's controlled by the block `[ItemView]` in the kvconfig file of the theme you use: Only inside that block, and provided that you have `text.margin=true` in it, change the values of `text.margin.top` and `text.margin.bottom`, and restart pcmanfm-qt. You could also change `frame.top` and `frame.bottom` in the same block, but that might make the theme ugly. However, the changes will be for _all lists in all Qt apps_.

If you mean the icon view or something else, it may depend on the Qt plugins of the DE you use. KDE may enforce large margins on icon views; LXQt doesn't.
